hazelcast/src/main/java/com/hazelcast/instance/impl/JetExtension.java

```diff
@@ -70,18 +67,34 @@ public void beforeStart() {
     }
 
     public void afterStart() {
-        if (node.isRunning() && node.getClusterService().getClusterVersion().isGreaterOrEqual(Versions.V5_0)) {
-            activated = true;
-            jetServiceBackend.getJobCoordinationService().startScanningForJobs();
+        if (tryActivate()) {
             logger.info("Jet is enabled");
```

Comment:
I think we should move this "Jet is enabled" log into `tryActivate`'s activation block.